controllers/grid/users/reviewer/form/AdvancedSearchReviewerForm.php

```diff
@@ -211,6 +211,21 @@ public function fetch($request, $template = null, $display = false)
             ]
         ]);
 
+        // Get the submission's authors
+        $publication = $this->getSubmission()->getCurrentPublication();
+
+        $authors = [];
+        foreach($publication->getData('authors') as $author) {
+            $affiliations = [];
+            foreach($author->getAffiliation(null) as $affiliationName) {
+                $affiliations[] = $affiliationName;
+            }
+
+            $authors[$author->getFullName()] = implode(',', array_filter($affiliations));
```

Comment:
What about trying to get data only in one (requested) locale and only then fallback to the first available?

To get requested locale you can use `PKP\facades\Locale::getLocale()`.
`Author::getFullName()` accepts a default locale as a 3rd argument, otherwise it uses Submission locale.